### PR TITLE
Rework rules

### PR DESCRIPTION
**Pull request by ammodev**
This pull request includes significant updates to the `Writerside/topics/rules/rules.md` file, focusing on clarifying and refining server rules for better readability and enforcement. The most important changes include modifications to the rules on griefing, game modifications, player behavior, and trading.

Clarifications and refinements:

* **Griefing**: Updated the language to clarify that any destruction or theft from other players will result in an immediate ban and emphasized the need for explicit permission for any changes or usage of private areas. [[1]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L21-L50) [[2]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L105-L131)

* **Game Modifications**: Revised the rules to specify that any unauthorized modifications, including those that automate player activities or bypass anti-cheat measures, are strictly prohibited, and clarified the consequences of using such modifications. [[1]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L21-L50) [[2]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L105-L131)

* **Player Behavior**: Enhanced the guidelines on respectful behavior, including detailed rules for voice chat usage and the handling of conflicts, emphasizing the importance of politeness and respect among players. [[1]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L66-R95) [[2]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L144-R180)

* **Boosting and IRL Trading**: Strengthened the prohibition on multiple accounts and clarified the restrictions on trading, including the consequences of exploiting server mechanics for personal gain. [[1]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L66-R95) [[2]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L144-R180)

* **Authority of Moderators and Admins**: Reiterated that moderators and admins have the final say in all matters, even if their decisions contradict the official rules, and requested players to comply without engaging in lengthy discussions. [[1]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L66-R95) [[2]](diffhunk://#diff-19685bad04cb7dde549f60435ca124af29b0e3af7f5672727d363e8f9192c633L144-R180)

### CODE REVIEW

File: Writerside/topics/rules/rules.md [hunk 1]
```diff
@@ -18,36 +18,32 @@ Solltet ihr vermehrt negativ mit Aktionen auffallen oder gegen die Regeln versto
 mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser findet ihr weiter unten.
 
 1. **Griefing wird zu keiner Zeit toleriert**. Sollte uns auffallen, dass du Gebäude oder Bauwerke
-   von anderen Spielern mutwillig zerstörst oder andere Spieler bestiehlst, droht ein sofortiger
-   Ausschluss aus dem Projekt.
+   von anderen Spielern zerstörst oder andere Spieler bestiehlst, droht ein sofortiger
+   Ausschluss vom Server.
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:22]:
```suggestion
   von anderen Spielern zerstörst oder andere Spieler bestiehlst, wirst du sofort
   vom Server ausgeschlossen.
```

File: Writerside/topics/rules/rules.md [hunk 2]
```diff
@@ -18,36 +18,32 @@ Solltet ihr vermehrt negativ mit Aktionen auffallen oder gegen die Regeln versto
...
-   hin, dass alles, was eine Tätigkeit des Spielers automatisiert, untersagt ist, egal ob es sich um
-   Minecraft-Modifikationen, externe Software, Exploits, Hardware oder sonstige Möglichkeiten
-   handelt. Modifikationen, welche das Ziel haben AntiCheat-Maßnahmen zu umgehen oder auszutricksen, sind ebenfalls
-   untersagt.
+2. **Spielmodifikationen, die keinen unfairen Vorteil gegenüber anderen Spielern verschaffen, sind erlaubt, solange sie
+   die Kommunikation zwischen Server und Client nicht verändern.** Wir weisen darauf hin, dass andere Modifikationen
+   und/oder Hacked-Clients nicht zulässig sind. Des Weiteren ist jegliche Form der Automatisierung einer
+   Spieleraktivität untersagt, unabhängig davon, ob es sich um Minecraft-Modifikationen, externe Software, Exploits,
+   Hardware oder sonstige Möglichkeiten handelt. Ebenso sind Modifikationen, die dazu dienen, Anti-Cheat-Maßnahmen zu
+   umgehen oder auszutricksen, nicht zulässig.
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:36]:
```suggestion
   Hardware oder sonstige Möglichkeiten handelt. Ebenso sind Modifikationen zur Umgehung oder Täuschung von Anti-Cheat-Maßnahmen nicht erlaubt.
```

File: Writerside/topics/rules/rules.md [hunk 3]
```diff
@@ -18,36 +18,32 @@ Solltet ihr vermehrt negativ mit Aktionen auffallen oder gegen die Regeln versto
...
+
+   Alleine das Betreten des Servers mit einer unerlaubten Modifikation wird bereits als Verstoß angesehen und kann zu
+   einem Ausschluss führen. Sollte es also nicht klar sein, ob eine Modifikation erlaubt ist, verwendet sie
+   nicht, [werft einen Blick auf die Liste geprüfter Mods][verified-mods] oder [kontaktiert den Support][support].
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:40]:
```suggestion
   Bereits das Betreten des Servers mit einer unerlaubten Modifikation gilt als Verstoß und kann zum
   Ausschluss führen. Wenn also unklar ist, ob eine Modifikation erlaubt ist, verwendet sie
   nicht, [schaut in die Liste geprüfter Modifikationen][verified-mods] oder [kontaktiert den Support][support].
```

File: Writerside/topics/rules/rules.md [hunk 4]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
    >
    {style="tip"}
 
-4. Es wird erwartet, dass ihr euch **freundlich und respektvoll gegenüber anderen Spielern**
-   verhaltet. Wir tolerieren **keine Belästigung anderer Spieler jeglicher Art**. Seien es
-   Beleidigungen, Anfeindungen, Scamming oder sonstiges Trolling.
+4. **Freundliches und respektvolles Verhalten wird erwartet.**  
+   Behandelt andere Spieler stets höflich und respektvoll. Jegliche Form von **Belästigung**, sei es durch
+   Beleidigungen, Anfeindungen, Scamming oder Trolling, wird nicht geduldet.
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:64]:
```suggestion
   Behandelt andere Spieler stets höflich und respektvoll. **Belästigungen jeglicher Art**, sei es durch
   Beleidigungen, Anfeindungen, Scamming oder Trolling, werden von uns nicht geduldet.
```

File: Writerside/topics/rules/rules.md [hunk 5]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
...
+      untersagt, wo es keinen sinnvollen Zweck erfüllt.
 
-   > Solltet ihr ein Problem mit einer anderen Person haben, versucht es **miteinander** zu
-   > lösen. Sollte dies nicht möglich sein, [kontaktiert bitte einen Moderator][support].
-   > Offener Streit wird nicht geduldet und wir behalten uns vor Spieler dafür temporär vom Server auszuschließen.
-   >
-   {style="tip"}
+        - **Erlaubt:** In klar definierten Events wie **Casinos**, **Musikevents**, **DJ-Clubs** oder anderen
+          Situationen, wo es ausdrücklich Sinn ergibt.
+        - **Verboten:** Überall sonst, insbesondere in alltäglichen Sprachchats oder um andere Spieler zu trollen.
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:72]:
```suggestion
        - **Verboten:** Überall sonst, insbesondere in alltäglichen Sprachchats oder zum Trollen anderer Spieler.
```

File: Writerside/topics/rules/rules.md [hunk 6]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
...
+
+   > Habt ihr ein Problem mit jemandem? Versucht zunächst, es **friedlich miteinander** zu klären. Falls das nicht
+   möglich ist, [wendet euch bitte an einen Moderator][support].  
+   > Offene Streitigkeiten werden nicht toleriert, und wir behalten uns vor, Spieler für solche Konflikte vorübergehend
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:76]:
```suggestion
   > Offene Streitigkeiten werden nicht toleriert, und wir behalten uns das Recht vor, Spieler wegen solcher Konflikte vorübergehend
```

File: Writerside/topics/rules/rules.md [hunk 7]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
...
+   auszuschließen.  
+   > {style="tip"}
+
+5. **Boosting und IRL Trading sind strikt untersagt.**
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:80]:
```suggestion
5. **Boosting und IRL-Trading sind strengstens verboten.**  
```

File: Writerside/topics/rules/rules.md [hunk 8]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
...
+        - **Verboten:** Überall sonst, insbesondere in alltäglichen Sprachchats oder um andere Spieler zu trollen.
+
+   > Habt ihr ein Problem mit jemandem? Versucht zunächst, es **friedlich miteinander** zu klären. Falls das nicht
+   möglich ist, [wendet euch bitte an einen Moderator][support].  
+   > Offene Streitigkeiten werden nicht toleriert, und wir behalten uns vor, Spieler für solche Konflikte vorübergehend
+   auszuschließen.  
+   > {style="tip"}
+
+5. **Boosting und IRL Trading sind strikt untersagt.**  
+   Jeder Spieler darf lediglich **einen Account** besitzen. Die Nutzung von Zweit-Accounts zum Farmen von Ressourcen,
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:81]:
```suggestion
   Jeder Spieler darf lediglich **einen Account** besitzen. Die Nutzung von Zweitaccounts zum Farmen von Ressourcen,
```

File: Writerside/topics/rules/rules.md [hunk 9]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
...
+   Geld oder anderen Vorteilen ist verboten.
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:82]:
```suggestion
   Geld oder anderen Vorteilen ist untersagt.
```

File: Writerside/topics/rules/rules.md [hunk 10]
```diff
@@ -63,27 +59,40 @@ mit einem Ausschluss vom Server rechnen. Eine detaillierte Auflistung dieser fin
...
+   erfolgen.
 
-5. **Boosting und IRL Trading ist nicht gestattet.** Jeder Spieler darf nur über **einen Account**
-   verfügen. Zweit-Accounts, um Geld oder sonstige Dinge zu farmen, sind nicht erlaubt. Außerdem darf
-   jeglicher Handel zwischen den Spielern nur mit auf dem Server verfügbaren Items
-   und [Währungen][economy] betrieben werden.
+   Darüber hinaus ist es nicht gestattet, dass Spieler oder deren Freunde dem Server lediglich beitreten, um in
+   beispielsweise einem Adventskalender Items zu droppen und anschließend sofort wieder zu disconnecten, ohne eine
+   tatsächliche Absicht zu haben, aktiv am Spielgeschehen teilzunehmen. Sollte es auffallen, dass Spieler und/oder deren
+   Freunde diese Mechaniken ausnutzen, wird es für beide in einem Ausschluss enden.
```
Comment (twisti-dev) [inline Writerside/topics/rules/rules.md:90]:
```suggestion
   Des Weiteren ist es nicht erlaubt, dass Spieler oder ihre Freunde den Server nur betreten,
   um beispielsweise Items aus einem Adventskalender an andere zu übergeben und dann sofort wieder disconnecten, ohne eine
   tatsächliche Absicht zu haben, aktiv am Spielgeschehen teilzunehmen. Sollte auffallen, dass Spieler und/oder deren
   Freunde diese Mechaniken ausnutzen, wird dies mit einem Ausschluss für beide geahndet.
```

**Review by twisti-dev — CHANGES_REQUESTED** [review]
Insgesammt so eine 8/10 👍

File: Writerside/topics/rules/rules.md (hunk 1 above)
Comment (Keviro) [reply Writerside/topics/rules/rules.md:22]: Ich finde `droht ein sofortiger Ausschluss vom Server` hört sich besser an ehrlich gesagt

Comment (twisti-dev) [reply Writerside/topics/rules/rules.md:22]: Bei "drohen" finde ich, dass es so klingt, als ob es nicht immer passiert - aber wir bannen ja immer

**Review by twisti-dev — APPROVED** [review]

**Pull request merged**